socs/agents/hwp_encoder/agent.py

```diff
@@ -720,6 +720,12 @@ def _stop_acq(self, session, params=None):
 
 
 def make_parser(parser=None):
+    """
+    Arguments:
+        port (int): Listening port of Agent for receiving UDP encoder packets. This 
+                    should match what is defined in the bbb encoder process configs
+        ip (str): IP of bbb running the corresponding encoder process
+    """
```

Comment:
Thanks for adding these. Can you move them to the `description` argument within the `pgroup.add_argument()` calls below. See [argparse docs](https://docs.python.org/3/library/argparse.html#description) for an example.